documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -94,6 +94,11 @@ Capacity limits can be set for the following broker resources, using the standar
 
 Because Strimzi Kafka brokers are homogeneous, Cruise Control applies the same capacity limits to every broker it is monitoring.
 
+[NOTE]
+====
+Cruise Control assumes brokers resources to be homogeneous. Therefore, it is recommended to set both `Kafka.spec.kafka.resources.limits.cpu` and `Kafka.spec.kafka.resources.limits.memory`, so that Kubernetes will ensure broker pods are limited to the same capacity. This is especially important when using nodes with heterogeneous resources (e.g. CPU cores).
```

Comment:
We for sure do not recommend setting memory limits much -> it is not guaranteed and can cause problems when the memory needs to be returned.

Reply:
Yeah, I mean, the full recommendation would be to set requests == limits. Should I add this point too?